org.techxartisan.openterfaceQT/org.techxartisan.openterfaceQT.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.techxartisan.openterfaceQT
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: openterfaceQT
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+
+modules:
+  - name: libusb
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26.tar.bz2
+        sha256: 12ce7a61fc9854d1d2a1ffe095f7b5fac19ddba095c259e6067a46500381b5a5
+    build-commands:
+      - ./configure --prefix=/app
+      - make
+      - make install 
```

Comment:
this is not how `buildsystem: autotools` work.  Not sure what documentation you followed here but it's incorrect.

Anyway use the shared-module.